README.md

```diff
@@ -18,8 +18,8 @@ This command will pull Hazelcast Docker image and run a new Hazelcast Instance.
 For the simplest end-to-end scenario, you can create a Hazelcast cluster with two Docker containers and access it from the client application.
 
 ```
-$ docker run -e JAVA_OPTS="-Dhazelcast.local.publicAddress=<host_ip>:5701" -p 5701:5701 hazelcast/hazelcast
-$ docker run -e JAVA_OPTS="-Dhazelcast.local.publicAddress=<host_ip>:5702" -p 5702:5701 hazelcast/hazelcast 
+$ docker run -e JAVA_OPTS="-Dhazelcast.local.publicAddress=<host_ip>:5701" -p 5701:5701 hazelcast/hazelcast:$HAZELCAST_VERSION
+$ docker run -e JAVA_OPTS="-Dhazelcast.local.publicAddress=<host_ip>:5702" -p 5702:5701 hazelcast/hazelcast:$HAZELCAST_VERSION
```

Comment:
I always think about simplifying this without using  `hazelcast.local.publicAddress`. 

The following should be fine if you disable smartrouting if you want to connect hazelcast cluster from your local network. 
```
$ docker run -d -p 5701:5701 hazelcast/hazelcast:3.12.6
$ docker run -d -p 5702:5701 hazelcast/hazelcast:3.12.6
```

Reply:
Yeah, but then we would need to explain how to disable smart client. I remember we added it for the reason that this was an issue. I remember David tried to run it, but it didn't work and then we added it.

So, to be frank I'd keep it that way.